multiselect.go

```diff
@@ -159,18 +174,20 @@ func (m *MultiSelect) OnChange(key rune, config *PromptConfig) {
 	// and we have modified the filter then we should move the page back!
 	opts, idx := paginate(pageSize, options, m.selectedIndex)
 
+	tmplData := MultiSelectTemplateData{
+		MultiSelect:   *m,
+		SelectedIndex: idx,
+		Checked:       m.checked,
+		ShowHelp:      m.showingHelp,
+		PageEntries:   opts,
+		Config:        config,
+	}
+
 	// render the options
-	m.Render(
-		MultiSelectQuestionTemplate,
-		MultiSelectTemplateData{
-			MultiSelect:   *m,
-			SelectedIndex: idx,
-			Checked:       m.checked,
-			ShowHelp:      m.showingHelp,
-			PageEntries:   opts,
-			Config:        config,
-		},
-	)
+	m.Render(MultiSelectQuestionTemplate, tmplData)
+
+	offset := computeCursorOffset(MultiSelectQuestionTemplate, tmplData, opts, idx, m.termWidthSafe())
+	m.OffsetCursor(offset)
```

Comment:
What do you think about adding a new method to `Renderer` called somethign like `RenderWithCursorOffset` or something that does these three lines together? It will help dry up the code base as well as give us a single place where accessibility is implemented in case we need to change things down the line.